<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentShopView</class>
 <widget class="QWidget" name="StudentShopView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>396</width>
     <height>209</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="viewShoppingCartOption">
      <property name="text">
       <string>View &amp;Cart</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="termLabel">
        <property name="text">
         <string>Term</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="quantityLabel">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="termSelectOption"/>
      </item>
      <item>
       <widget class="QPushButton" name="addTextbookToCartOption">
        <property name="text">
         <string>&amp;Add Textbook to Cart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="quantityOption"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="itemsListView"/>
    </item>
    <item>
     <widget class="QPushButton" name="viewDetailsOption">
      <property name="text">
       <string>View Content Details</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
